### DISCUSSION TITLE
Setting labels for custom menus / commands?

### DISCUSSION THREAD

**Discussion by MattCheely** [helix-editor/helix#1744] [Q&A]
I recently realized I can add my own menu entries to do things like create "sub-menus", but the labeling gets a bit awkward. As an example if I do something in my config file like:

```
[keys.normal.space]
"f" = { "f" = "file_picker", "s" = ":write", "c" = ":open ~/.config/helix/config.toml" }
```
I can use `space f` to get into file mode, but the `f` entry after pressing `space` has no label, and the `c` entry displays the open command, but it would be nice to make it say something like "Edit Helix config".  Are either of those possible?

**Comment by hervyqa** [comment]
> [TheAwiteb#1](https://github.com/TheAwiteb/helix/pull/1) has a solution...
> 
> The config would look like this:
> 
> ```toml
> [keys.normal.space]
> "i: Soft wrap toggle" = ":toggle-option soft-wrap.enable"
> ```
